packages/monaco-editor-wrapper/src/wrapper.ts

```diff
@@ -155,17 +155,17 @@ export class MonacoEditorLanguageClientWrapper {
         this.init(userConfig);
 
         // Always dispose old instances before start
-        this.editor?.disposeEditor();
-        this.editor?.disposeDiffEditor();
+        this.editorApp?.disposeEditor();
```

Comment:
Do we really need to have this be named `editorApp` now, is `editor` not sufficient, or is there something special about specifying it this way (similar to the renaming of the class instance as well, trying to understand the motivation there).

Reply:
Will come with next update